index.js

```diff
@@ -1,7 +1,11 @@
 const awaitEvent = require('await-event')
 const socketLocation = require('socket-location')
 
-module.exports = async function httpRequestToUrl (request) {
+module.exports = async function httpRequestToUrl (request, opts = {}) {
+  // default to true
+  opts.followProxies = Object.prototype.hasOwnProperty.call(opts, 'followProxies')
```

Comment:
This felt awkward, but [all hail standard linting](https://eslint.org/docs/rules/no-prototype-builtins).

Reply:
This could just be `opts.followProxies = !!opts.followProxies`.

Reply:
I had some time to come back to this today -- I'm not sure I follow your feedback here. Changing this to `opts.followProxies = !!opts.followProxies` resulted in a test failing (I pushed the change and left it, that's why Travis is red right now)

If `opts.followProxies` is present, we want to use its truth-y value.  For this case `opts.followProxies` = !!opts.followProxies` works well.

However, if `opts.followProxies` is _undefined_ (meaning the value isn't set) we want the function to behave as though `opts.followProxies` was true.  We want to follow proxies by default.  

My proposal would be to go back to the 

```
opts.followProxies = Object.prototype.hasOwnProperty.call(opts, 'followProxies')
    ? opts.followProxies : true
``` 

code.  However, I don't have strong options here. If you don't like this code I'm happy to make whatever changes you to w/r/t to handling the default case.  Just let me know.

(Also, in case it wasn't obvious, there's no sense of urgency on this one.  We're taking a step back from this particular module -- but it's probably still worth getting in for other folks who are using this)

Reply:
You could do `opts.followProxies = opts.followProxies === true`. That should do what you had in mind without needing to care about prototypes or calls.

Reply:
I could, and I have. Tests pass, resolving comment